src/runtime/policy/policy.ts

```diff
@@ -221,17 +226,21 @@ export class PolicyField {
       }
     }
 
+    if (allowedUsages.length === 0) {
```

Comment:
when the policy is stringified, will this default allowedUsage be written? if so, maybe we should add the default at the parser level?

Reply:
Nope, it will only write out the explicitly listed annotations.